projects/storefrontapp-e2e-cypress/cypress.ci.json

```diff
@@ -0,0 +1,12 @@
+{
+  "defaultCommandTimeout": 15000,
```

Comment:
It's opinionated thus optional, but I would prefer to keep the same conditions for CI and non-CI test runs if possible, to have consistent results (we have 10 seconds for non-CI version).